### PR TITLE
	modified:   Common/Model/HttpSubscription.cs

### PR DESCRIPTION
**Pull request by johns397**
modified:   Common/Model/Subscriptions.cs
	modified:   Hub/Rules/SubscriptionValidator.cs
	modified:   Tests/IntegrationTests.cs
	modified:   WebSubClient/Hubs/WebSubClientHub.cs
	modified:   WebSubClient/Rules/HubSubscriptions.cs

### CODE REVIEW

File: Tests/IntegrationTests.cs [hunk 1]
```diff
@@ -69,16 +71,33 @@ public class IntegrationTests : IDisposable {
             return port;
         }
 
+        private void VerifiySubscription(Subscription sentSubscription, Subscription returnedSubscription)
+        {
+            Assert.Equal(sentSubscription.Secret, returnedSubscription.Secret);
+            Assert.Equal(sentSubscription.Mode, returnedSubscription.Mode);
+            Assert.Equal(sentSubscription.Topic, returnedSubscription.Topic);
+            // Server isn't setting the HubURL; should it?
```
Comment (wdvr) [inline Tests/IntegrationTests.cs]: I think it shouldn't - the hub knows it is the hub.

**Comment by johns397** [comment]
Wouter,   Yes, but it could be the case that the hub that where the client sends the subscription request has a load balancer and hands off the request to a separate system with a different HubURL.    Having the hub report back the HubURL would enable this capability.
  John


-----Original Message-----

**Comment by wdvr** [comment]
wouldn't the URL be the load balancer's URL then? I would think this is transparent to the client, the client always talks to the load balancer.

**Comment by wdvr** [comment]
I'll merge this one in. Let's figure out the hubURL parameter as part of https://github.com/fhircast/sandbox/issues/53.

**Pull request merged**